src/Display/Popover/Popover.tsx

```diff
@@ -56,7 +56,11 @@ class Popover extends React.Component<Props, State> {
         aria-busy="false"
         tabIndex={0}
       >
-        <PopoverChildren onClick={this.handleOnClick}>
+        <PopoverChildren
+          className="popover-children"
+          data-testid="children"
```

Comment:
why do we need `className="popover-children"` if we have `data-testid="children"`?